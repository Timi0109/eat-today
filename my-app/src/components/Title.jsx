import React from "react";
import styles from "./Title.module.css";

const Title = () => {
    return <h2 className={styles.title}>EatToday</h2>
}


export default Title;