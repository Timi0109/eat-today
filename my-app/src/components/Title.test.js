import React from "react";
import Title from "./Title";

describe("Title tests", () => {
  it("renders correctly", () => {
    const { container } = render(<Title></Title>);
    expect(container).toMatchSnapshot();
  });

});
