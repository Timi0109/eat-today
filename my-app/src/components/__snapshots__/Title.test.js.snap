// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title tests renders correctly 1`] = `
<div>
  <h2
    class="title"
  >
    EatToday
  </h2>
</div>
`;
